stonesoup/sensormanager/base.py

```diff
@@ -111,7 +111,8 @@ def choose_actions(self, tracks, timestamp, nchoose=1, **kwargs):
             Time at which the actions are carried out until
         nchoose : int
             Number of actions from the set to choose (default is 1)
-
+        return_reward: bool
+            Whether to return the reward for chosen actions (default is False)
         Returns
         -------
         : dict
```

Comment:
Could you add something here to describe how the output changes if return_reward=True? I.e. it returns a list of dictionaries and the reward (as a numpy array usually of 1D?).